screens/unauth/TestScreen.tsx

```diff
@@ -0,0 +1,5 @@
+import {View} from 'react-native';
+
+export default function TestScreen() {
+  return <View></View>;
+}
```

Comment:
I think this should be removed 😄

Reply:
I think i can leave it. Sometimes when i maka a screen or compoenent it is faster to do it on test screen than navigate to desired screen and there develop

Reply:
Sure, no problem :)